src/js/common/components/AdditionalInfo.jsx

```diff
@@ -10,7 +10,8 @@ export default class AdditionalInfo extends React.Component {
     this.state = { open: false };
   }
 
-  toggle = () => {
+  toggle = (e) => {
+    if (e) e.preventDefault();
```

Comment:
Instead of doing this, we could also add `type="button"` to the button element. It's treated as submit button without that attribute.

Reply:
Thanks @jbalboni, it looks like the button element already has type="button".

Reply:
@cehsu The close button doesn't.

Reply:
Thanks again for clarifying @jbalboni, that makes sense.

Reply:
Thanks for your recommendation @jbalboni, the proposed change does not prevent the automatic/unintentional selection.
<img width="934" alt="screen shot 2017-12-07 at 12 02 52 pm" src="https://user-images.githubusercontent.com/16051172/33736230-aed0ce24-db46-11e7-9cde-997d5832ee40.png">
<img width="913" alt="screen shot 2017-12-07 at 12 03 12 pm" src="https://user-images.githubusercontent.com/16051172/33736232-b0bfbd80-db46-11e7-9c4c-bf3195872709.png">

Reply:
Ah, I see. In that case, the way it was is fine, assuming it's focusing on the link again when you close it.

Reply:
It's not focusing on the link again, now that you mention it, nor does it with the button role assigned. How about I pull https://github.com/department-of-veterans-affairs/vets.gov-team/issues/6045 into this work so that there's no more need for the close button, rather than writing logic to set the focus?

Reply:
Sounds good to me